jax/api.py

```diff
@@ -1046,7 +1082,8 @@ def f_pmapped(*args, **kwargs):
         global_axis_size=axis_size,
         devices=tuple(devices) if devices is not None else devices,
         name=flat_fun.__name__,
-        mapped_invars=tuple(axis is not None for axis in in_axes_flat))
+        mapped_invars=tuple(axis is not None for axis in in_axes_flat),
+        donate=donate)
```

Comment:
Would it make sense to call this donate_invars (following mapped_invars)?

Reply:
I feel like semantically it is very close to static_argnums (e.g. it identifies arguments by index that are donated). Is there a reason to prefer invars over argnums?

Reply:
My thinking was that unlike static_argnums, donate is carried through the Jaxpr and it describes the behavior of the invars.

Reply:
Have kept this as `donate_argnums` on the API and `donate_invars` on the primitive.